### ISSUE TITLE
Render markdown

### ISSUE THREAD

**Issue by sedrubal** [m1ga/pusteblume#18]
Hi,

it would be nice, if the app would render markdown...

**Comment by m1ga** [comment]
if you want to help you could create some regex so I can include them here:
https://github.com/m1ga/pusteblume/blob/master/app/alloy.js
but I'll have a look too

**Comment by sedrubal** [comment]
I'm no regex guru, but I think something like
```
.+\n={3,}\n
```

should be a `<h1>` and

```
.+\n-{3,}\n
```
should be a `<h2`>, but I don't know how to replace strings by using parts of them...

Maybe you can use this: https://gist.github.com/jbroadway/2836900

I'm currently quite busy, so I can't help you ;)

**Comment by m1ga** [comment]
hm, h1 is
~~~
\#{3}(.+)\n 
~~~
when I have a look at https://diasporafoundation.org/formatting
You can test it when you use http://regexpal.com/ 

I'll add some more later

**Comment by sedrubal** [comment]
Oh I think this is for
```
###Header
```
But I also know
```
Header
======
```
I don't know if this syntax is allowed in diaspora, too.

**Comment by sedrubal** [comment]
Oh I think this is for

```markdown
###Header
```

But I also know

```markdown
Header
======
```

I don't know if this syntax is allowed in diaspora, too.

**Comment by m1ga** [comment]
ok, the ==== seems to work too even its not listed in the markdown help :)

![styles](https://cloud.githubusercontent.com/assets/4334997/6092139/4b88acd8-aed0-11e4-9fc1-5a08046a5955.jpg)

~~~
headline: #,##,###
bold: **,__
italic:  *,_
bold italic: ***,___
~~~

have to adjust the headlines but it seems to work

**Comment by sedrubal** [comment]
Cool, thank you.

BTW. Do you maintain the f-droid repo?

**Comment by m1ga** [comment]
no, not yet. Have to include it in the readme since many people are asking for it :smile: hopefully in the near future I'll add a private repo you could include in your fdroid

**Comment by m1ga** [comment]
Will add some more but for now more elements are working:
https://github.com/m1ga/pusteblume/releases/tag/1.25

**Comment by sedrubal** [comment]
thank you, I didn't think, that you'll react as quick :D

But I think there is a new Problem: There is a post like this:

```markdown
Hello World,
----------------
blablablablabla
```

And in the detail view I only see:

```markdown
Hello World,
---------------
```
And this is not rendered...

**Comment by m1ga** [comment]
think that is not related to the markdown. In the comment-view I only display two rows otherwise the postings would take all the space

**Comment by sedrubal** [comment]
Ah ok.

**Comment by m1ga** [comment]
But I found an error in the comments code since I've changed the row creation when I added the slideshow function. Will be fixed soon :tm:

**Comment by m1ga** [comment]
try 1.27 please: https://github.com/m1ga/pusteblume/releases/tag/1.27
 Hopfully last bugfix for today :smile:

**Comment by sedrubal** [comment]
Can the app display gifs? Recently I saw following unrendered markdown:

```
![](http://example.com/cat.gif)
```

**Comment by sedrubal** [comment]
And I often see this:

```markdown
![Image/photo](http://url.to/my/cat.PNG ""]
```

And I saw a normal link (`http://en.wikipedia.org/cats`) which was not clickable.
